### PR TITLE
Bluecoat updates - Fixes

### PR DESCRIPTION
**Pull request by centralscrutiniser**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

- Fixed inc file typos
- Fixed SQL insert typos
- Added missing definitions.inc.php entries

### CODE REVIEW

File: sql-schema/250.sql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 INSERT INTO `graph_types`(`graph_type`, `graph_subtype`, `graph_section`, `graph_descr`, `graph_order`) VALUES ('device',  'sgos_client_connections',  'network',  'HTTP Client Connections',  0);
 INSERT INTO `graph_types`(`graph_type`, `graph_subtype`, `graph_section`, `graph_descr`, `graph_order`) VALUES ('device',  'sgos_server_connections',  'network',  'HTTP Server Connections',  0);
 INSERT INTO `graph_types`(`graph_type`, `graph_subtype`, `graph_section`, `graph_descr`, `graph_order`) VALUES ('device',  'sgos_client_connections_active',  'network',  'HTTP Client Connections Active',  0);
-INSERT INTO `graph_types`(`graph_type`, `graph_subtype`, `graph_section`, `graph_descr`, `graph_order`) VALUES ('device',  'sgos_server_connections_active',  'network',  'HTTP Server Connections Acitve',  0);
```
Comment (murrant) [inline sql-schema/250.sql]: You will have to add a new schema file to correct this.

Comment (centralscrutiniser) [reply sql-schema/250.sql]: @murrant OK, will get that sorted. Thanks.

File: sql-schema/252.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+DELETE FROM * `graph_types` WHERE `graph_descr` = 'HTTP Server Connections Acitve';
```
Comment (laf) [inline sql-schema/252.sql]: Should be named `251.sql` :)

Comment (centralscrutiniser) [reply sql-schema/252.sql]: Sorry @laf got confused with another branch I'm working on.

**Comment by centralscrutiniser** [comment]
@laf Anything else I need to do on this?

**Comment by laf** [comment]
Just waiting on scrutinizer checks now.

**Pull request merged**